js/constants/config.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     authUrl: (userId) => `${vaultHost}/v1/users/${userId}`,
     replacementUrl: adHost
   },
-  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd'
+  braveExtensionId: 'mnojpmjdmbbfmejpflffifhffcmidifd',
+  PDFJSExtensionId: 'ckglimbbooghmggmkikhbolcfmgniemo'
```

Comment:
with this change, pdfjs doesn't load for me at all (same thing that was happening for you before this change). i.e., chrome-extension://ckglimbbooghmggmkikhbolcfmgniemo/https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf is blank but chrome-extension://adnmjfhcejodgpaljdmlmjoclihpcfka/https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf loads fine. maybe this origin is unique per install?